run.pxi

```diff
@@ -26,6 +57,49 @@
   []
   (p/describe @p/*project*))
 
+(defn loadns [ns]
+  ;(print (str "(require " ns ")"))  
+  (eval (read-string (str "(require " ns ")"))))
+
+(defn showdoc [ns function]
+  
+  ;loads other possible namespaces in case function is in other namespace
+  (def name (str (if (not= ns "") (str ns "/") "")))
```

Comment:
The function (loadns) was a lot more complicated initially. I'll just drop the function.